state-chain/runtime/src/chainflip.rs

```diff
@@ -941,3 +941,19 @@ impl FetchesTransfersLimitProvider for SolanaLimit {
 		})
 	}
 }
+
+pub struct EvmLimit;
+impl FetchesTransfersLimitProvider for EvmLimit {
+	fn maybe_transfers_limit() -> Option<usize> {
+		Some(50)
+	}
+
+	fn maybe_ccm_limit() -> Option<usize> {
+		// For ccm calls we don't batch
+		None
```

Comment:
We don't batch but it might be worth limiting this anyway since each ccm requires a signature ceremony... will think about it.

Reply:
I've opened an issue to come back to this.